src/webextension/firstrun/components/app.css

```diff
@@ -5,7 +5,11 @@
 body {
   padding: 0;
   margin: 0;
+  color: #0c0c0d;
   background-color: #f9f9fa;
+  font: caption;
+  font-size: 13px;
+  -moz-user-select: none;
```

Comment:
Should we pre-emptively add the non-prefixed version as well?

Reply:
I'm not entirely convinced we event want `user-select` due to its unexpected side-effects (unable to highlight/select content from entries). What do you think @jimporter?

Whatever you all decide I still think this is OK to ship as-is since we are just running it in Firefox for now and it's still marked "experimental". Good eye, though @pdehaan.

Reply:
We do, but we want `<FieldText/>` to enable selection. Since the spec is still a draft though, I'm not sure how much value there is in adding the non-prefixed version. There's really no guarantee it works like the moz-prefixed one, and it's already changed in some ways from how browsers implement it.